packages/x-components/src/x-modules/search/components/__tests__/promoted.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { Promoted as PromotedModel } from '@empathyco/x-types';
+import { mount, Wrapper } from '@vue/test-utils';
+import Vue from 'vue';
+import { createPromotedStub } from '../../../../__stubs__/promoted-stubs.factory';
+import { installNewXPlugin } from '../../../../__tests__/utils';
+import { getXComponentXModuleName, isXComponent } from '../../../../components/x-component.utils';
+import Promoted from '../promoted.vue';
+
+function renderPromoted({
```

Comment:
We don't need to create a wrapper component to test it as it's a simple component without slots. But it's not mandatory to change it. 

The criteria or the recommendation to use it or not is in the glossary:

- Sometimes it's easier to create a wrapper component in which we can import and render our
  component or components, especially when we use slots, defining the template. It's preferable to
  test the component isolated without this wrapper. Notice that you can also include this within a
  factory function in order to override the template as another property.

Reply:
won't do